.github/workflows/build-package-test.yml

```diff
@@ -82,7 +83,9 @@ jobs:
           PACKAGING_IMAGE_PLATFORM: "${{matrix.TARGET_PLATFORM}}"
 
       - name: Push images
-        run: ./ci/push_images
+        run: |
+            export CURRENT_BRANCH=${GITHUB_REF##*/}
```

Comment:
what was that CURRENT_BRANCH for ?

Reply:
For determining the current branch to be able to compare it in push images if it is develop branch or not